common/src/stack/command/stack/commands/list/attr/plugin_host_intrinsic.py

```diff
@@ -81,14 +82,19 @@ def run(self, scope_mappings):
 						# the attr os.version is '{major_version}.x'
 						# release is now '{OS}{major_version}'
 						if pallet_os in rel:
-							os_version = f'{rel.replace(pallet_os, "")}.x'
+							major_version = rel.replace(pallet_os, "")
+							os_version = f'{major_version}.x'
```

Comment:
> Question - for versions with a dot, would we rather see the leading dot or not

I think both the dot in minor_version and trailing `.x` on major_version are unnecessary. Just my two cents.

Reply:
So just:

os.version -> `15`
os.minor_version -> `sp2`

?

I think I like that, but I'm hesitant to change os.version right now, since that one does get used in a few places in stacki and tdc-infra (and possibly even Snapi, I'd think?).  Maybe after we cut 5.6...

Reply:
Yep. Just like that. And Snapi doesnt use the os.version attr. Without the minor version, snapi cant determine the bootaction for sles12 sp3 vs sp2. So other, more hardcoded, methods are used.

But 
```
os.version -> 15
os.minor_version -> sp2
```
is kinda just aesthetics so either way is good.